.github/ISSUE_TEMPLATE/03-jsrt-bug.yaml

```diff
@@ -0,0 +1,37 @@
+name: JsRT Bug
```

Comment:
Maybe using "API" instead of "JsRT" in this and the next template is going to be a bit more user-friendly, we want the list of templates to be clear on what category the request falls in. Not very strong opinion here.

Reply:
@ShortDevelopment Please could you make this change (API instead of JsRT) then we may merge this.

Reply:
Sure.
I'll come back to this on tuesday (when I'm home again)

Reply:
Done